core/src/main/scala/cats/Monad.scala

```diff
@@ -155,7 +155,7 @@ import simulacrum.{noop, typeclass}
 
     def step(branches: Branches): F[Either[Branches, A]] =
       branches match {
-        case (cond, conseq) :: tail =>
+        case cond, conseq :: tail =>
```

Comment:
Um 🧐

Reply:
Looks like a quite severe bug in ScalaFmt